source/docs/yearly-changes/yearly-changelog.rst

```diff
@@ -0,0 +1,163 @@
+         .withVelocityY(-joystick.getLeftX() * MaxSpeed) // Drive left with negative X (left)
+         .withRotationalRate(-joystick.getRightX() * MaxAngularRate) // Drive counterclockwise with negative X (left)
+         )
+      );
+
+      joystick.a().whileTrue(drivetrain.applyRequest(() -> brake));
+      joystick.b().whileTrue(drivetrain
+         .applyRequest(() -> point.withModuleDirection(new Rotation2d(-joystick.getLeftY(), -joystick.getLeftX()))));
+
+      if (Utils.isSimulation()) {
+         drivetrain.seedFieldRelative(new Pose2d(new Translation2d(), Rotation2d.fromDegrees(90)));
+      }
+
+      drivetrain.registerTelemetry(logger::telemeterize);
+   }
+
+.. image:: images/swerve-simulation-video.gif
+   :alt: GIF showing swerve simulation support
+
+.. important:: Swerve API requires all necessary swerve devices to be v6 devices. e.g. 4 drive TalonFX, 4 steer TalonFX, 1 Pigeon 2.0, 4 CANcoders.
+
+Signal Logging
+^^^^^^^^^^^^^^
+
+We've added a comprehensive signal logger API (Java, C++, Python) that represents a real-time capture of signals for supported devices. Signal logging can be useful for analysis of signals over a period of time. In applications, they can be useful for tuning PID gains, characterization of systems, analyzing latency on a system and much more.
+
+Information on configuring the logger and extracting the logs are available in the API Reference and Tuner sections.
+
+.. image:: images/tuner-x-log-extractor.png
+   :alt: Log extractor page in Tuner X
+
+Signal API Improvements
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Users can now disable signals by setting their update rate to 0Hz. To reduce boilerplate when disabling signals, we have added an ``optimizeBusUtilization`` function that takes a list of signals as a parameter. This would automatically disable all signals that are not used.
+
+Setting a given signal's frequency behavior has been improved by keeping track of the signal with the highest frequency in a frame. The highest frequency's signal is used for the frame instead of the most recent signal.
+
+Additionally, the following new functions have been added.
+
+* ``refreshAll``
+
+  * refreshes all passed in signals
+
+* ``getAppliedUpdateFrequency``
+
+  * retrieves the actual update frequency of a given signal
+
+New Motion Magic Controls
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For a full list of new motion magic controls, see the controls API documentation (Java, C++, Python).
+
+Differential Mechanisms
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important:: ``DifferentialMechanism`` requires both Pro and CANFD. ``SimpleDifferentialMechanism`` is a lower performance alternative that requires neither.
+
+``DifferentialMechanism`` (Java, C++, Python) provides an easy way to control two-axis differential mechanisms such as two motor elevators (motor on the left and right side of the elevator but is not mechanically linked). Information on this is available here.
```

Comment:
```suggestion
``DifferentialMechanism`` provides an easy way to control two-axis differential mechanisms such as two motor elevators (motor on the left and right side of the elevator but is not mechanically linked). Information on this is available here.
```
We don't have python support yet, so I'd remove the language list (and we kinda have c# support already, both will be fleshed out later, so it's not really important to give a language list)